crates/kitsune_p2p/types/src/reverse_semaphore.rs

```diff
@@ -89,6 +89,7 @@ impl Drop for ReverseSemaphorePermit {
 mod tests {
     use super::*;
 
+    // @maackle: note, this is flaky, it has failed for me locally once
```

Comment:
Ignore then?

Reply:
Well, I opted not to, because it seemed like such a rare fluke of a failure that I couldn't reproduce. I feel it's very unlikely that it will fail for anyone else ever again, but if it does, at that point we can remove it. This was like a "first warning".

It failed only because it's based on sensitive timing and my system was doing a bunch of other stuff at the time (including running all other tests).

Reply:
Funny, it just failed on CI now too. Ignoring!